code/modules/jobs/job_types/head_of_personnel.dm

```diff
@@ -56,3 +60,10 @@
 	if(locate(/datum/holiday/ianbirthday) in SSevents.holidays)
 		undershirt = /datum/sprite_accessory/undershirt/ian
 
+//only pet worth reviving
+/datum/job/hop/get_mail_goodies(mob/recipient)
+	. = ..()
+	// Strange Reagent if the pet is dead.
+	for(var/mob/living/simple_animal/pet/dog/corgi/ian/staff_pet in GLOB.dead_mob_list)
+		. += list(/datum/reagent/medicine/strange_reagent = 20)
+		break
```

Comment:
Pls

Reply:
I don't know if this is a bad thing or what you're commenting on, sorry.

Reply:
if you need to find a single instance like this you can use locate()